src/Components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import 'bootstrap/dist/css/bootstrap.css';
+
+import './SideBar.scss';
+
+/**
+ * SideBar component
+ * @returns {void}
+ */
+class SideBar extends Component {
+  state = {
+    checked: true,
+    display: true,
```

Comment:
I think the property of displaying the nav shouldn't come directly from the state of the component because the action to trigger this behavior will come from a different component

Reply:
Thanks @dusmel